docs/install-config/run-installer-script.md

```diff
@@ -41,7 +41,7 @@ sudo ./install.sh --with-trivy
 ```
 
 For more information about Trivy, see the [Trivy documentation](https://github.com/aquasecurity/trivy).
-For more information about how to use Trivy in an webproxy environment see [Configure custom Certification Authorities for trivy](administration/vulnerability-scanning/configure-custom-certs.md)
+For more information about how to use Trivy in an webproxy environment see [Configure custom Certification Authorities for trivy](../administration/vulnerability-scanning/configure-custom-certs.md)
```

Comment:
are you sure?

I tried to reach that on the preview it doesn't wokr 

https://deploy-preview-621--goharbor-io.netlify.app/docs/main/install-config/run-installer-script/administration/vulnerability-scanning/configure-custom-certs.md

Reply:
I suggest using https://gohugo.io/content-management/cross-references/

Reply:
Actually the syntax is working well, there's another issue, the file i've modified is pointing to this link :
https://deploy-preview-621--goharbor-io.netlify.app/docs/edge/install-config/run-installer-script/